### ISSUE TITLE
Integrate the ontology diagram generator

### ISSUE THREAD

**Issue by edmondchuc** [RDFLib/pyLODE#37]
Integrate the ontology diagram generator into pylode.

Repository: https://github.com/edmondchuc/ontology_diagram_generator

Proposal: An additional optional argument can be passed into the pylode command-line application to automatically generate a PlantUML-based class diagram of the ontology.

**Comment by smarie** [comment]
On this topic of generating the diagram 

 - this example in the readme has a neat diagram: https://raw.githack.com/agldwg/agrif-ont/master/agrif.html#overview . Do you know with which library it was generated ? The look and feel is great.

 - a great addition could be to make a link with interactive webvowl navigation, for example when the user clicks on the diagram. I opened a thread on webvowl to ask for help there: https://github.com/VisualDataWeb/WebVOWL/issues/169

**Comment by graybeal** [comment]
Something to keep in mind with respect to this request is that pyLODE may be used on ontologies of all sizes. before deciding which ontology generator to use it may be worth looking at how well it handles large ontologies. (I know that's not very concretely helpful but I wanted to note the consideration at least.)

**Comment by smarie** [comment]
For reference, Here is an illustration of the Webvowl approach: https://doc.realestatecore.io/3.1.3/core.html

**Comment by charlesvardeman** [comment]
The ontology diagram at  https://raw.githack.com/agldwg/agrif-ont/master/agrif.html#overview looks like it was generated by [CMap](https://cmap.ihmc.us/). Once upon a time, there was a CMap Ontology Edition (COE) but I don't think that is alive any longer. The source for the Webvowl approach is in the https://github.com/RealEstateCore/doc repository. Here is [the script](https://github.com/RealEstateCore/doc/blob/master/build.py) that does the processing to generate the Webvowl visualization.

**Comment by smarie** [comment]
yes @charlesvardeman for our project I made a variant of the script you mention, it works perfectly. Note however that it requires that `owl2vowl.jar` (and therefore a local JVM) and the whole webvowl skeleton folder to be present. So this is maybe not something that you would like to embed into pyLODE, rather some recipe that you might wish to explain in the doc.
